### DISCUSSION TITLE
Preview should take you to the screen you are designing.

### DISCUSSION THREAD

**Discussion by poirazis** [Budibase/budibase#7619] [Ideas]
When you are in the BB builder and you hit preview, you should be taken to the preview of the screen you are designing and not the home page.

Now this raises some complexities, what if the route needs parameters ? Well, if there are default preview parameters defined on the screen as described here #7479 they should be respected, or random ones will be picked ( similarly to the in builder preview )

**Comment by JayP718** [comment]
Agreed, this isn't a major issue for us but it would be nice. Definitely way more intuitive, being taken to the screen you are currently designing. We always create a dummy quick link so that way we can go back and forth.
